buddyboss/buddyboss-platform
# PROD-667 Fix Activity Header space issue at the right side

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3222 

### How to test the changes in this Pull Request:

1. Go to Activity
2. Check space between activity header title and more options dots

### Proof Screenshots or Video
https://ibb.co/j8Wr9DR

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
